tests/Oryx.Integration.Tests/NodeEndToEndTests.cs

```diff
@@ -646,6 +646,43 @@ public async Task Node_SoundCloudNgrxApp()
                 });
         }
 
+        [Theory]
+        [MemberData(nameof(TestValueGenerator.GetNodeVersions), MemberType = typeof(TestValueGenerator))]
+        public async Task Node_NodeSassExampleSample(string nodeVersion)
```

Comment:
Did you run this integration test locally?

Reply:
yes took me 47 minutes, tried with intermediate directory but rsync is failing .. checking why